.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,36 +1,34 @@
 ---
-title: 'Bug report'
+name: 🐛 Bug report
+about: Something's not quite right? You're in the right place!
 labels: user report
 ---
 
-### 🐛 Bug Report
+# 🐛 Bug Report
 
-<!-- Please answer these questions before submitting your issue. Thanks! -->
+<!-- 
+  Please fill out each section below before submitting your 🐛 bug report.
 
-#### What version of Wrangler are you using (`wrangler -V`)?
+  Before opening a new issue, please:
+  * search for existing issues: https://github.com/cloudflare/wrangler/issues
+  * make sure you are using the latest release: https://workers.cloudflare.com/docs/quickstart/updating-the-cli/
+  
+  Thanks! -->
 
-<pre>
-$ wrangler -V
+## Description
```

Comment:
"Description" is not super necessary here; the following fields will gather all the info we need. I'd remove this field.